erpnext/stock/report/stock_ledger/stock_ledger.py

```diff
@@ -240,3 +250,16 @@ def get_item_group_condition(item_group):
 			item_group_details.rgt)
 
 	return ''
+
+
+def get_package_tag_batch_details():
```

Comment:
I believe we are moving this out of stock_ledger.py and to a package_tag.py file inside of erpnext